registry/test/text-color.test.ts

```
@@ -14,30 +14,58 @@ import {
 
 testStoryConsistency('text-color')
 
-testStory('text-color', () => {
+testStory({
+  story: 'text-color',
+}, () => {
   it('change and clear text color via inline menu', async ({}) => {
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)